style.css

```diff
@@ -1 +1,129 @@
 /*  Add your styling here */
+
+body{
+    text-align: center;
```

Comment:
Are you sure you want all text on your entire page to be center aligned? It's usually best to avoid this sort of specific rule on the `body`